ios/Exponent/Versioned/Core/Api/Components/GoogleMaps/AIRGoogleMapHeatmap.h

```diff
@@ -4,7 +4,7 @@
 //  Created by David Cako on 29 April 2018.
 //
 
-#import <Google-Maps-iOS-Utils/GMUHeatmapTileLayer.h>
```

Comment:
nit: not sure if that's upstream intentional, but i feel like on ios the `#import <Module/Header.h>` format is more correct.